deploy/olm-catalog/windows-machine-config-operator/manifests/service-monitor.yaml

```diff
@@ -0,0 +1,23 @@
+# Prometheus Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: windows-machine-config-operator
+  name: windows-machine-config-operator-metrics
```

Comment:
Shouldn't this be `windows-exporter`? The confusion here is that you are introducing this here as `windows-machine-config-operator-metrics` and you then have a constant in the same commit that says `windows-exporter` and then you rename `windows-machine-config-operator-metrics` to `windows-exporter` in another commit.